### ISSUE TITLE
Add sound

### ISSUE THREAD

**Issue by jbphet** [phetsims/resistance-in-a-wire#180]
The sound design for this simulation is in a state where we can now implement it in the simulation.  The sound design document can be found [here](https://docs.google.com/document/d/1eof0--X1w6NBm5HWdMll8i3HvbOsOCE_D_Tu1WoooQc/edit#) and a working prototype version that uses phet-io and wrappers is available [here](https://phet-dev.colorado.edu/html/phet-io-wrapper-sonification/1.0.0-dev.69/phet-io-wrapper-sonification/resistance-in-a-wire/html/resistance-in-a-wire-sonification.html?sonificationFile=sonificationOptions&resistanceSoundSetting=marimbaDuringInteraction&noSliderSounds&accessibility).

**Comment by jbphet** [comment]
@emily-phet and @Ashton-Morris - I've added an initial implementation of sound to RIAW, please try it out and log any feedback in this issue.

Link: https://phet-dev.colorado.edu/html/resistance-in-a-wire/1.6.0-dev.3/phet/resistance-in-a-wire_en_phet.html?supportsSound

**Comment by jbphet** [comment]
One thing I noticed on this version, and it is consistent with the prototype sound design, is that if you start the sim up and move one of the sliders down slowly with the mouse, it moves a ways before a sound occurs, but if you move it up, a sound occurs right away.  I'm thinking it would be more intuitive if sound was generated at the same distance from the center position.

**Comment by Ashton-Morris** [comment]
I agree that the trigger points for the sliders should be the same going up or down. Currently there are 6 sound points if you slide it up and 5 if you slide it down on each slider. Maybe changing the number of points to 12 total might help?

The current reset sound is different from the what we are using in the JT sonification sim, for continuity we should use the same one.

**Comment by emily-phet** [comment]
@jbphet @Ashton-Morris Let's discuss this in a meeting on 10/23 (google invite already sent).

**Comment by emily-phet** [comment]
@jbphet @Ashton-Morris 
A few thoughts:

- A note on trigger points for the tone when using the slider - looks like it triggers each 0.1 for resistivity, each 2 for L, and 1.5?  - plus triggering quickly after the 'up' motion wherever you are. @jbphet can you describe exactly how this is implemented - what is the trigger tied to? The value of the slider + something related to timing/direction of initial movement?
- When the resistance sound is the deepest - (resistivity = 1.0, A = 0.1) move the L slider back and forth from top to midway. In this region it seems like there's something extreme and slightly uncomfortable happening with the sound. On my computer with no headphones, it sounds like a high tinny part of the deep resistance tone. Mac with headphones, there's a little speaker crackle.  If others are experiencing this we might want to discuss. If it's just my speaker setup, we can ignore - it's probably won't be noticed by most students/teachers, and if noticed it's not really 'bad'.

**Comment by emily-phet** [comment]
@emily-phet Going to check in with AP and KP about publishing RIAW without enhanced options (marimba sound as default).
Also to check with KP on licensing question.

**Comment by emily-phet** [comment]
@jbphet 
@ariel-phet thinks RIAW is good to go sound-wise. Waiting to hear back from @kathy-phet 
AP also mentioned that the resistance word is moving on the screen now that there are three decimal places. He would like that word fixed in place. Is that something you can do?

**Comment by emily-phet** [comment]
@jbphet @Ashton-Morris 
KP and AP have given their feedback - the sound is good to go. They would like something done to minimize the amount of 'jumping around' the resistance word does as the resistance value changes. Then it's good to go to move on to the next steps toward publication.

**Comment by jbphet** [comment]
In [version 1.6.0-dev.3](https://phet-dev.colorado.edu/html/resistance-in-a-wire/1.6.0-dev.3/phet/resistance-in-a-wire_en_phet.html?supportsSound) there is an issue with when the sliders generate sounds.  In this version, moving a slider up or down any amount after initial reload generates a sound.  Once this initial motion has occurred for a given slider, if the sim is then reset, moving the slider up any amount causes a sound, but moving it down requires a bit of distance before any sound occurs.  

@Ashton-Morris and @emily-phet and I discussed this in last week's sound design review meeting and decided that it should be revised.  I've updated the behavior such that a little bit of slider motion is required to get the first sound, the amount of distance is equivalent in the up and down direction, and the behavior before and after a reset is consistent.

@Ashton-Morris and @emily-phet - please review [version 1.6.0-dev4](https://phet-dev.colorado.edu/html/resistance-in-a-wire/1.6.0-dev.4/phet/resistance-in-a-wire_en_phet.html?supportsSound) and let me know if this is what we're after.

**Comment by jbphet** [comment]
@emily-phet tried out the version in the comment just above and said that the number of bins and the other aspects of the behavior seemed correct.

**Comment by jbphet** [comment]
Here is a dev version with sound turned on (i.e. no query parameter required), bins improved, and a proposed solution in place for the issue where distortion was occurring.  @Ashton-Morris - please test this and let me know if you feel like any last minute adjustments to volume or any other sound behavior is needed.  If not, I'll create a release candidate (RC) and start the testing process.

**Comment by jbphet** [comment]
The sound design has been approved, and RC testing is underway.  Closing.
